3.10/aql/functions-arangosearch.md

```diff
@@ -1161,3 +1163,84 @@ FOR doc IN viewName
   SORT doc.text, TFIDF(doc) DESC
   RETURN doc
 ```
+
+Search Highlighting Functions
+-----------------------------
+
+{% include hint-ee.md feature="Search highlighting" %}
+
+### OFFSET_INFO()
+
+`OFFSET_INFO(doc, paths) → offsetInfo`
```

Comment:
`paths` can also be just a string, then it'll be treated the same as an array of only 1 string

Reply:
Hm, okay. Too many syntax variants tend to be confusing, though.